src/components/interaction-designer/blocks/BlockToolbar.vue

```diff
@@ -7,13 +7,13 @@
           v-if="isBlockSelected"
           v-b-tooltip.hover="trans('flow-builder.deselect-block')"
           :icon="['far', 'check-circle']"
-          class="fa-btn text-info"
+          class="cursor-pointer text-info"
```

Comment:
`fa-btn` wasn't applied for some reason.
Can someone please check and post below where that css class is located?

Reply:
I couldn't find `fa-btn` either. It appears it should come from Font Awesome, but there are no references there.

Reply:
Thanks!